### ISSUE TITLE
Home env/Docker detection on windows

### ISSUE THREAD

**Issue by nunzioono** [jpmcb/nvim-llama#8]
Since HOME is not defaultly set on windows, nvim running ollama.lua doesn't find HOME variable and it doesn't run.

![image](https://github.com/jpmcb/nvim-llama/assets/36959525/cd630b43-353a-477a-ae87-e68776600038)

The error showed is this one:

![image](https://github.com/jpmcb/nvim-llama/assets/36959525/cae12fc2-c614-4a73-a6ed-23850581e35e)

**Comment by jpmcb** [comment]
Good call: let me boot up my windows box and give this a spin - thanks for raising this issue!

**Comment by nunzioono** [comment]
Thank you to create this awesome plugin, if you like it i can make a pull request tomorrow with the fixes needed

**Comment by nunzioono** [comment]
I'm trying to come up with an explaination for which the plugin doesn't recognize docker either. But nothing for now:

![image](https://github.com/jpmcb/nvim-llama/assets/36959525/ceaeb432-5ae6-42fb-b0fb-c6d5acb53455)

Even if docker is for sure installed and tested on the same machine, running the same function in the lua playground works differently from running it inside init.lua

![image](https://github.com/jpmcb/nvim-llama/assets/36959525/bcb530a5-35e1-4782-8e9e-cba7eb24a94a)

**Comment by jpmcb** [comment]
> Thank you to create this awesome plugin, if you like it i can make a pull request tomorrow with the fixes needed

Of course! It's my pleasure!

> I'm trying to come up with an explaination for which the plugin doesn't recognize docker 

I should have some bandwidth this weekend to take a peek. My guess is the plugin runs in a different space that doesn't have the `Docker` executable in it's same runtime.
